README.md

```diff
@@ -199,6 +202,9 @@ Applies single operation object `operation` on `document`.
 - `validateOperation` Whether to validate the operation, or to pass a validator callback
 - `mutateDocument` Whether to mutate the original document or clone it before applying
 
+It modifies the `document` object and `operation` - it gets the values by reference.
+If you would like to avoid touching your values, clone them: `jsonpatch.apply(document, jsonpatch.deepClone(operation))`.
```

Comment:
I guess it should be `applyOperation`. Not being pedantic but `apply` is deprecated.